go/cmd/vpnkit-expose-port-controller/vpnkit-expose-port-controller.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: vpnkit-expose-port-controller
+      namespace: kube-system
+      labels:
+        name: vpnkit-expose-port-controller
+  - apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRole
+    metadata:
+      name: vpnkit-expose-port-controller
+      labels:
+        name: vpnkit-expose-port-controller
+    rules:
+      - apiGroups: [ v1 ]
+        resources: [ services ]
+        verbs: [ watch ]
+  - apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRoleBinding
+    metadata:
+      name: vpnkit-expose-port-controller
+      namespace: kube-system
+      labels:
+        name: vpnkit-expose-port-controller
+    roleRef:
+      kind: ClusterRole
+      name: vpnkit-expose-port-controller
+      apiGroup: rbac.authorization.k8s.io
+    subjects:
+      - kind: ServiceAccount
+        name: vpnkit-expose-port-controller
+        namespace: kube-system
+  - apiVersion: apps/v1beta2
+    kind: DaemonSet
+    metadata:
+      name: vpnkit-expose-port-controller
+      namespace: kube-system
+      labels:
+        name: vpnkit-expose-port-controller
+    spec:
+      selector:
+        matchLabels:
+          name: vpnkit-expose-port-controller
+      updateStrategy:
+        type: RollingUpdate
+      template:
+        metadata:
+          labels:
+            name: vpnkit-expose-port-controller
+        spec:
+          serviceAccount: vpnkit-expose-port-controller
+          tolerations:
+            - effect: NoSchedule
+              operator: Exists
+          containers:
+            - name: app
+              image: 'docker/kubernetes-vpnkit-expose-port-controller:current'
```

Comment:
thats not actually the image...

Reply:
What should we call this image?

Reply:
I think we should put it under `linuxkitprojects` for now